Patterns/Chinese/Chinese-DateTime.yaml

```diff
@@ -179,8 +179,9 @@ DateTimePeriodTillRegex: !simpleRegex
   def: (?<till>到|直到|--|-|—|——)
 DateTimePeriodPrepositionRegex: !simpleRegex
   def: (?<prep>^\s*的|在\s*$)
-HourRegex: !simpleRegex
-  def: (?<hour>00|01|02|03|04|05|06|07|08|09|0|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|1|2|3|4|5|6|7|8|9)
+HourRegex: !nestedRegex
+  def: \b{BaseDateTime.HourRegex}

```

Comment:
any differences if add this \b? I don't see the previous logic for this.

Reply:
I've added this for consistency with the other languages. All tests passed.